### ISSUE TITLE
[Screen Reader - Elixir - Review Settings] Screen reader does not associate the column header with their row cell on table under 'Review Settings' tab.

### ISSUE THREAD

**Issue by mohit406** [microsoft/fluentui#17886]
**User Experience:**
Screen reader user would be affected if screen reader doesn't associate column header with their respective cell value as he/she won't be able to understand the context of table.

Note: User credentials should NOT be included in the bug.

**Repro Steps:**

1. Open URL: https://elixirppe.microsoft.com/home#  and login with valid creds.
2. Navigate to Settings button at the bottom of screen and select it.
3. Navigate to all controls under Review settings tab.
4. Verify that screen reader is reading column header along with the row elements or not.

**Actual Result:**
Screen reader is not announcing column header along with the row elements on table under 'Review Settings' tab, due to this user will not get information association of items with the column headers.

**Expected Result:**
Screen reader should announce column header along with the row elements on table under 'Review Settings' tab.

**Note:**

- This issue exists with Narrator & NVDA.
- Same issue exists throughout the application where this type of control appears.
- Same issue repro with all tables throughout application.

**MAS Reference:**
[MAS 1.3.1 - Info and Relationships](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B97358E88-4367-4121-BF9D-F0C6597D03B7%7D&file=MAS%201.3.1%20%E2%80%93%20Info%20and%20Relationships%20.docx&wdLOR=cC67AE285-FA8E-4A9C-8615-42813289BF4A&action=default&mobileredirect=true&cid=8b8b5d44-71df-4efd-a034-0a931254a86f)

**Reference Links:**

1. [Accessibility Insights! ](https://apc01.safelinks.protection.outlook.com/?url=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906834622&sdata=0WbVE4dkxMrMm9bbxE1O5xshTfo1%2B0frnzlbmWVeHTs%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier.
2. External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this [Link](https://apc01.safelinks.protection.outlook.com/?url=https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906839611&sdata=D5fhtDtz6f9O9hGT6oPoowXq0L/TCfaejH73thazOvA%3D&reserved=0). “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
3. Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process related queries.

**Comment by mohit406** [comment]
A11yMAS, A11ySev2, Accessibility, AILimited, Benchmark, Feature-HCL-Elixir-Feb21, HCL, MAS1.3.1, Win10-Chrome

**Comment by mohit406** [comment]
This bug is copied from: https://msazure.visualstudio.com/One/_workitems/edit/9370641

**Comment by claire-MSFT** [comment]
@tomi-msft Any ETA in regards to when this bug will be fixed? We have a dependency on this bug and would love to have this resolved sooner per accessilbity requirements.

**Comment by tomi-msft** [comment]
Hi @mohit406, thanks for filing the issue. After some initial investigation, I was unable to repro the bug in Narrator, I believe the column headers are announcing as expected. Here are the steps I followed:

- Turned on Narrator (WinKey + Ctrl + Enter)
- Made sure Narrator is in Scan Mode (CAPS LOCK + SPACE)
- Navigate to the Review Settings tab in Elixir
- Navigate to the nearest table (T)
- Navigate through the table (Ctrl + Alt + Arrow Keys)
- Notice that the header is announced when you move from row cell to row cell
